src/layout.rs

```diff
@@ -308,6 +310,7 @@ impl Layout {
             },
             constraints: Vec::new(),
             expand_to_fill: true,
+            prefer_equal_chunks: false,
```

Comment:
Addressed in https://github.com/ratatui-org/ratatui/pull/395/commits/17790b7fbd2a3f9252c0f8b6b63f4560fd904920. Let me know if you have better suggestions for naming.

Reply:
What about something like AllocateLeftoverSpace (and have this incorporate the expand to fill option, which is pub(crate) so can be removed without breaking anything)
```rust
enum AllocateLeftoverSpace {
  None,
  Last,
  Equal,
}
```
(I'm never quite sure whether to add an explicit `None` vs using Option though)

Reply:
> (I'm never quite sure whether to add an explicit None vs using Option though)

personally, if it is a enum similar to this, then i (as a user) would prefer to just have to define `AllocateLeftoverSpace::None` and `AllocateLeftoverSpace::Equal` instead of `(Option::)None` and `Some(AllocateLeftoverSpace::Equal)`

also as for a techincal aspect, this *could* save one WORD(bytes) of space, because of the option identification